docs/index.rst

```diff
@@ -6,50 +6,27 @@ Scripture Burrito Documentation
 
 .. image:: ../logo/burrito_logo.png
 
-This is the official documentation for the evolving Scripture Burrito standard. See the development `milestones here <https://github.com/bible-technology/scripture-burrito/milestones>`_).
+After several years of development and testing, we are pleased to announce the availability of **Scripture Burrito 1.0-rc** specification! We recommend that developers of Scripture and Scripture-related applications test and adopt this specification for interchanging data with other systems. Out of the box Scripture Burrito is designed to support the following types of data:
 
-**This is 1.0.0-beta2** based on `JSON Schema <https://json-schema.org/>`_.
+- :ref:`scripture_text_flavor`
+- :ref:`scripture_print_flavor`
+- :ref:`scripture_audio_flavor`
+- :ref:`scripture_sign_language_flavor`
+- :ref:`scripture_braille_flavor`
+- :ref:`scriptural_text_stories_flavor`
 
```

Comment:
These are candidate flavors, we need two independent, interoperable implementations of a given flavor before a particular flavor will be added to 1.0.   We should say that clearly.

The goal is to have interoperability, if we have a proliferation of flavors that don't interoperate, that's a bad thing, not a good thing.